src/directives/colony/clearRoom.ts

```diff
@@ -83,6 +93,22 @@ export class DirectiveClearRoom extends Directive {
 				log.notify(`Removing clearRoom directive in ${this.pos.roomName}: operation completed.`);
 				this.remove();
 			}
+		// Clear path if controller is not reachable
+		} else if (this.room && this.room.creeps.length > 1) {
+			let currentlyDismantling = _.find(this.room.flags, function(flag) {
+				return (flag.color == DirectiveDismantle.color && flag.secondaryColor == DirectiveDismantle.secondaryColor)
```

Comment:
use `DirectiveDismantle.find(this.room.flags)` to get a list of all dismantling directives in the room